src/llmcompressor/transformers/finetune/training_args.py

```diff
@@ -32,6 +32,12 @@ class TrainingArguments(HFTrainingArgs):
             )
         },
     )
+    per_device_oneshot_batch_size: int = field(
```

Comment:
Why per device? Considering gptq's sequential nature/one active execution device

Reply:
This name is just to match the existing `per_device_train_batch_size` argument name. We can alias this or resolve `per_device_train_batch_size = per_device_oneshot_batch_size = batch_size` in the future

Reply:
Given that oneshot is unlikely to support device-parallel computation in the future, I'm fine using a more concise name now

Reply:
Renamed to `oneshot_batch_size` to allow users to have separate batch sizes for oneshot and train. We can add a batch_size later if we think that's a better interface

Reply:
Renamed to `calibration_batch_size`